specification/containerstorage/resource-manager/Microsoft.ContainerStorage/preview/2023-07-01-preview/containerstorage.json

```diff
@@ -444,13 +431,13 @@
         "x-ms-long-running-operation": true
       }
     },
-    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerStorage/pools/{poolName}/volumeSnapshots": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerStorage/pools/{poolName}/snapshots": {
```

Comment:
>snapshots [](http://example.com/codeflow?start=126&length=9)

Existing customers who are using previous preview version, aren't these breaking changes to them? How are you ensuring the breaking experience for them?

Reply:
There aren't any existing customers as the RP is still in development and it hasn't been made available anywhere public. We're hoping we can remove the `2023-03-01` version, so that our first publicly accessible release will use `2023-07-01`.

Reply:
As Simon says, 2023-03-01 is a test version and isn't being used by customers, so we want to remove it, so we don't have to support an unused version. Is there any chance we can do so in this PR?

Reply:
Thank you for sharing the details. You also have breaking changes review required label on your PR. Please get a breaking changes approval from Azure Breaking Changes Reviewers <azbreakchangereview@microsoft.com> and ARM can review the changes afterwards. Once you have the approval, please remove the "ARMChangesRequested" label from the PR to make the PR visible to the ARM reviewers.